Scripts/DCS-BIOS/doc/developerguide.adoc

```diff
@@ -119,142 +120,55 @@ If you feed the export stream data you receive from DCS-BIOS to the `processChar
 == The DCS-BIOS Lua Code
 
 DCS-BIOS is loaded by executing the `BIOS.lua` file.
-That file loads all other DCS-BIOS code. It also defines the hook functions for DCS export (`LuaExportStart` and friends), which mostly call functions in `Protocol.lua`.
+That file loads all other DCS-BIOS code.
 
 The following is an overview of the other files and their purpose:
 
-BIOSConfig.lua:: This file is loaded last, so it can override any settings that are defined in global variables.
-It has to set the variable `BIOS.protocol.io_connections` to a list of suitable connection objects to define where the export data gets sent to and how DCS-BIOS listens for commands.
+BIOSConfig.lua:: This contains TCP/UDP settings
 
-lib/ProtocolIO.lua:: This file contains classes that handle the actual data transmission, i.e. creating and using sockets.
+BIOS.lua:: Sets up the export hooks to DCS and makes calls to write all modules to JSON.
 
-lib/Protocol.lua:: The code in this file manages the list of known aircraft modules.
-Every frame, the `BIOS.protocol.step()` function in this file is called.
-That function checks what aircraft is currently being used and ensures that the correct data is exported.
+lib/io:: Contains all files needed for TCP/UDP transmission.
 
-lib/Util.lua:: This file includes a lot of utility classes and functions.
-Most of them implement the `MemoryMap` class and related classes or provide shortcuts to quickly define controls in aircraft modules.
+Protocol.lua:: Writes modules to memory and JSON.
 
-lib/A-10C.lua, lib/UH-1H.lua, etc:: Each aircraft module has its own file where all of its controls are defined.
+Module.lua:: This is the main class which all aircraft modules inherits from. Contains all functions needed for creating input and export functions for a control.
+
+lib/modules/air_craft/A-10C.lua, lib/modules/air_craft/UH-1H.lua, etc:: Each aircraft module has its own file where all of its controls are defined.
 
 
 == Adding a New Aircraft Module
 
-This section will describe how to add support for another aircraft module.
+This section describes how to add support for another aircraft module.
 
 === Export Modules in DCS-BIOS
 
-DCS-BIOS consists of several export modules (they are what you select in the "module" drop-down field in the control reference documentation). Each export module is assigned to one or multiple aircraft and several export modules can be active at the same time.
+DCS-BIOS consists of several export modules (they are what you select in the "module" drop-down field in `Bort` or `BIOSBuddy`. Each export module is assigned to one or multiple aircraft and several export modules can be active at the same time.
```

Comment:
Should we add links to these applications?

Reply:
Fixed